jina/executors/decorators.py

```diff
@@ -136,292 +74,13 @@ def arg_wrapper(self, *args, **kwargs):
     return arg_wrapper
 
 
-def _get_slice(
-        data: Union[Iterator[Any], List[Any], np.ndarray], total_size: int
-) -> Union[Iterator[Any], List[Any], np.ndarray]:
-    if isinstance(data, Dict):
-        data = islice(data.items(), total_size)
-    else:
-        data = data[:total_size]
-    return data
-
-
-def _get_size(data: Union[Iterator[Any], List[Any], np.ndarray], axis: int = 0) -> int:
-    if isinstance(data, np.ndarray):
-        total_size = data.shape[axis]
-    elif hasattr(data, '__len__'):
-        total_size = len(data)
-    else:
-        total_size = None
-    return total_size
-
-
-def _get_total_size(full_data_size, batch_size, num_batch):
-    batched_data_size = batch_size * num_batch if num_batch else None
-
-    if full_data_size is not None and batched_data_size is not None:
-        total_size = min(full_data_size, batched_data_size)
-    else:
-        total_size = full_data_size or batched_data_size
-    return total_size
-
-
-def _merge_results_after_batching(
-        final_result, merge_over_axis: int = 0, flatten: bool = True
-):
-    if not final_result:
-        return
-
-    if isinstance(final_result[0], np.ndarray):
-        if len(final_result[0].shape) > 1:
-            final_result = np.concatenate(final_result, merge_over_axis)
-    elif isinstance(final_result[0], list) and flatten:
-        final_result = list(chain.from_iterable(final_result))
-
-    return final_result
-
-
-def batching(
```

Comment:
seems batching is still imported in some tests, should they be removed/modified?

Reply:
in which tests? yes they should